plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -14,8 +14,7 @@ Level="Level"
 
 BFrames="Max B-frames"
 
-VAAPI.Codec="VAAPI Codec"
-VAAPI.Device="VAAPI Device"
+VAAPI.Device="VA-API Device"
```

Comment:
The official wiki uses VAAPI, while other/older sources use VA-API. I'm not sure there's merit in making this string change unless there is some official consensus that one is correct and one is not.